### PR TITLE
Python

### PR DESCRIPTION
**Pull request by srinu634**
Closing the 2 issues:
python / sklearn wrapper #4
tests for python wrapper #85

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,6 +25,8 @@ env:
     # test three different compilers
     #- CXX='g++-5' CC='gcc-5' BUILD_TYPE='release'
     
-install: sh ci/.travis_install.sh
+install: 
+    - sudo apt-get update
```
Comment (lacava) [inline .travis.yml]: why was this added?

File: python/setup.py [hunk 2]
```diff
@@ -50,7 +51,7 @@
        sources = ["pyfeat.pyx"],    # our cython source
        include_dirs = ['../src/',eigen_dir,shogun_include_dir]
                       +eigency.get_includes(include_eigen=False),
-       extra_compile_args = ['-std=c++14','-fopenmp','-Wno-sign-compare',
+       extra_compile_args = ['-std=c++1y','-Wno-sign-compare',
```
Comment (lacava) [inline python/setup.py]: -fopenmp should be added back. we just want to disable it on lpc

File: .travis.yml (hunk 1 above)
Comment (srinu634) [reply .travis.yml]: Hi Bill,
the system on travis was running on ubuntu 14.04. I added this statement as cython installation failed initially as it had an olde version of cython. I can remove it now as I don't think it's required anymore........

File: python/setup.py (hunk 2 above)
Comment (srinu634) [reply python/setup.py]: sure. I'll add it now

**Comment by srinu634** [comment]
#85 
#117
